### PR TITLE
Binary Audio/Image Post

### PR DESCRIPTION
**Pull request by tap2k**
* Added tests for Binary Image / Audio Post
* Added file_path_orig to the API for accessing original media files
* Added support for WAV files
* Make datatype API read only
Addresses Issue #37

### CODE REVIEW

File: apps/site/api/serializers/audio_serializer.py [hunk 1]
```diff
@@ -5,15 +5,15 @@
 from localground.apps.site import models
 
 
-class AudioSerializer(GeometrySerializer):
-    file_name_orig = serializers.CharField(required=False, read_only=True)
```
Comment (vanwars) [inline apps/site/api/serializers/audio_serializer.py]: I'm wondering if this field should be deleted. When files are uploaded, it's nice to be able to see the original file name.

File: apps/site/models/datatype.py [hunk 2]
```diff
@@ -10,8 +9,8 @@ class DataType(Base):
     DECIMAL = 5
     RATING = 6
 
-    name = models.CharField(max_length=255)
-    sql = models.CharField(max_length=500)
+    name = models.CharField(max_length=255, editable=False)
+    sql = models.CharField(max_length=500, editable=False)
```
Comment (vanwars) [inline apps/site/models/datatype.py]: When you make model changes that are also schema changes, the codebase also needs the corresponding Django migrations (which allow us to easily update the schema on our already-existing database, Django-style). To do this:

```shell
$ cd /localground/apps
$ python manage.py makemigrations
$ python manage.py migrate
$ git add site/migrations #don't forget to add the new migration file to the repo
```
Please add the migration file and push it to this PR.

**Comment by vanwars** [comment]
Tap,

This looks great! As soon as you add the migrations files, I will merge this into master and close this PR.

**Comment by tap2k** [comment]
I think I pressed the wrong button that said "Merge Pull Request" - I thought that meant added my most recent commit. Doh!

**Comment by tap2k** [comment]
Ill think about allowing binary in put / post - feels a bit off to me. Can we do this in the UI currently - swap out just the media file?

**Comment by vanwars** [comment]
OK. Please add your migration commit to the API_TAP branch, commit, push,
and then create another pull request.

**Comment by vanwars** [comment]
I agree -- feels weird. We should just decide and then ensure that we're
enforcing it on the /photos/<id>/ endpoint. There is no current support for
that in the UI.

**Comment by tap2k** [comment]
I think I already did this... Didn't I?

**Comment by vanwars** [comment]
Ah. Yes. I see it.

**Pull request merged**